crates/runtime/src/capability/builtin.rs

```diff
@@ -188,13 +188,26 @@ impl PartialEq for ActorIdentifier {
 
 impl Eq for ActorIdentifier {}
 
+#[derive(Clone, Debug, Eq, PartialEq)]
+/// Interface target to be invoked over `wRPC`
+pub struct InterfaceTarget {
```

Comment:
I think _probably_, I was trying to consider a future where the interface target isn't invoked over wRPC, and it felt better to constrain the target to an interface rather than a protocol. I had WrpcTarget first and I don't mind changing back